src/infrastructure/input.cpp

```diff
@@ -102,3 +108,16 @@ mfem::Vector FromInlet<mfem::Vector>::operator()(const axom::inlet::Table& base)
   }
   return result;
 }
+
+serac::input::BoundaryConditionInputInfo FromInlet<serac::input::BoundaryConditionInputInfo>::operator()(
+    const axom::inlet::Table& base)
+{
+  serac::input::BoundaryConditionInputInfo result;
+  // Build a set with just the values of the map
+  auto bdr_attr_map = base["attrs"].get<std::unordered_map<int, int>>();
+  for (const auto& [_, val] : bdr_attr_map) {
```

Comment:
Is the underscore a special character for an ignored variable? I can't find a reference on this.

Reply:
It's a popular convention in Python for ignoring something when unpacking a tuple, and it's used here as `std::ignore` doesn't work in structured bindings as it does in `std::tie`, which is a similar context.  For reasons I do not understand a standardized way of ignoring in a structured binding was not part of the C++17 standard.

Reply:
Since this smells like a language feature (that I agree probably should have been implemented), I think a name for this variable would be useful to keep modern C++ newbies like myself from getting confused.

Reply:
I think this is my only outstanding issue. It's not a big deal so I could be convinced otherwise. Any thoughts @white238 ?

Reply:
`_` is a pretty common convention. I am fine with `_` or `unused` if that's more clear.